File: server/src/main/kotlin/com/sggnology/server/db/sql/repository/ResourceInfoRepository.java
package com.sggnology.server.db.sql.repository;

import com.sggnology.server.db.sql.entity.ResourceInfo;
import com.sggnology.server.db.sql.entity.UserInfo;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ResourceInfoRepository extends JpaRepository<ResourceInfo, Long> {
}

File: server/src/main/kotlin/com/sggnology/server/db/sql/repository/ResourceInfoRepository.kt
package com.sggnology.server.db.sql.repository

import com.sggnology.server.db.sql.entity.ConfigInfo
import com.sggnology.server.db.sql.entity.ResourceInfo
import org.springframework.data.jpa.repository.JpaRepository

interface ResourceInfoRepository : JpaRepository<ResourceInfo, Long>
